Cargo.toml

```diff
@@ -65,6 +65,7 @@ elf = []
 macho = []
 pe = ["coff"]
 wasm = ["wasmparser"]
+xcoff = []
```

Comment:
This needs to be added as a dependency of other features as appropriate. For now, please add it to `read` and `doc`.

Reply:
Addressed. Thx!